<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HologramDepthmapClass</class>
 <widget class="QMainWindow" name="HologramDepthmapClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hologram Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbGen">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>50</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Generate Hologram</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="hologramdepthmap.qrc">
      <normaloff>:/HologramDepthmap/Resources/hologram_generation.png</normaloff>:/HologramDepthmap/Resources/hologram_generation.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRecon">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>50</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Reconstruct Image</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="hologramdepthmap.qrc">
      <normaloff>:/HologramDepthmap/Resources/hologram_reconstruction.png</normaloff>:/HologramDepthmap/Resources/hologram_reconstruction.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="rbCPU">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>    C P U</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbGPU">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>    G P U</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>299</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="hologramdepthmap.qrc"/>
 </resources>
 <connections/>
</ui>
